### PR TITLE
Local File Stream Support

### PR DESCRIPTION
**Pull request by noname22**
Supports streaming from from a local file stream server instead of a file through the lfsc_load command.

### CODE REVIEW

File: src/ipcstream.cpp [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+		if(s != LFSC_SOK)
+			throw StreamEx("could not get file flags");
+		
+		int size = FF_INPUT_BUFFER_PADDING_SIZE + 1024 * 32;
+		buffer = (unsigned char*)av_mallocz(size);
+		if(buffer == NULL)
+			throw StreamEx("failed to allocate RAM");
+
+		ctx = GenAVIOContext(buffer, size, /*(flags & LFSC_SERR_WRITE_PIPE) ? true : false*/ false);
```
Comment (eklann) [inline src/ipcstream.cpp:59]: What about the out-commented code?

File: src/player.cpp [hunk 2]
```diff
@@ -249,24 +250,49 @@ void Player::Run(IPC& ipc)
...
+
+					if(type == "load"){
+						FileStreamPtr fs = FileStream::Create();
+						fs->Open(message, false);
+						s = fs;
+					}
+
+					else {
+						s = lfsc->Open(Tools::StrToWstr(message));
+						s->Seek(0, SEEK_SET);
```
Comment (eklann) [inline src/player.cpp:271]: Is this needed on LFSC stream? If so, why?

Comment (noname22) [inline src/player.cpp:271]: Not entirely sure what you're referring to, specifically, but "load" creates a FileStream that loads from disk and "lfsc_load" creates an IpcStream that loads from a pipe/lfsc.

Comment (eklann) [inline src/player.cpp:271]: s->Seek(0, SEEK_SET);

**Pull request merged**